### PR TITLE
Reading DynamoDB Secret name from parameter

### PR DESCRIPTION
**Pull request by rajusem**
For osa gremlin service, we require to use different secret name "aws-dynamodb-osa", so reading it from parameter with default value "aws-dynamodb" so existing service don't need to change anything. For osa-gremlin-http service will pass DYNAMODB_SECRET_NAME value as  "aws-dynamodb-osa" into saas-analytics repo.

### CODE REVIEW

**Review by arajkumar — APPROVED** [review]
lgtm. But wait for @yzainee's  approval before merging.

**Comment by yzainee-zz** [comment]
LGTM

**Pull request merged**
